VimeoNetworking/VimeoNetworking/AuthenticationController.swift

```diff
@@ -211,6 +211,13 @@ final public class AuthenticationController
         self.authenticate(request: request, completion: completion)
     }
     
+    public func appTokenExchange(accessToken accessToken: String, completion: AuthenticationCompletion)
```

Comment:
Can we document this method? Pull in recent changes from `master` to see the format